### PR TITLE
[Testing] - User should be able to update progress through progress bar

### PR DESCRIPTION
**Pull request by Pratiyushkumar**
### Description - 
Previously, the user needed to go to my site and update his task progress.

Fix -

Now user can go to his task and click on the update progress button and after that, through the slider, he can update his progress and click on the save progress button to save the progress of the task and only the assignee can do it.

Parent PR is merged, this PR is regarding the test cases for the Preant PR

### Test Coverage:
image
image
image

### CODE REVIEW

File: src/utils/getPercentageOfDaysLeft.ts [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
-function inputParser(input: string) {
-    const parsedDate = moment(new Date(parseInt(input, 10) * 1000));
-    return parsedDate;
-}
-
 const getPercentageOfDaysLeft = (startedOn: string, endsOn: string): number => {
-    const startDate = inputParser(startedOn);
-    const endDate = inputParser(endsOn);
+    const startDate = moment(new Date(parseInt(startedOn, 10) * 1000));
```
Comment (vvaibhavdesai) [inline src/utils/getPercentageOfDaysLeft.ts:4]: can we take this piece out as a function

Comment (Pratiyushkumar) [reply src/utils/getPercentageOfDaysLeft.ts:4]: I didn't understand, why this needs to be taken out  as a separate function.

File: __tests__/Unit/Components/Tasks/ProgressIndicator.test.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const DEFAULT_PROPS = {
+    percentCompleted: 50,
+    startedOn: '10 july 2023',
+    endsOn: '14 July 2023',
+};
+
+describe('Progress Indicator', () => {
+    test('should render the ProgressIndicator', () => {
+        const { container } = render(<ProgressIndicator {...DEFAULT_PROPS} />);
+        console.log('container', container);
```
Comment (sahsisunny) [inline __tests__/Unit/Components/Tasks/ProgressIndicator.test.tsx:13]: ?

File: __tests__/Utils/handleProgressColor.test.ts
Comment (sahsisunny) [inline __tests__/Utils/handleProgressColor.test.ts:1]: Rremove the comments please

File: src/utils/getPercentageOfDaysLeft.ts (hunk 1 above)
Comment (sahsisunny) [reply src/utils/getPercentageOfDaysLeft.ts:4]: Before your changes, it was a separate function. Do you were facing any issues?

File: src/utils/getPercentageOfDaysLeft.ts [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+     * number of days left
+     */
     const totalDays = endDate.diff(startDate, 'days');
     const daysLeft = endDate.diff(new Date(), 'days');
 
-    // It provides the percentage of days left
+    /**
+     * It provides the percentage of days left
+     */
+
```
Comment (sahsisunny) [inline src/utils/getPercentageOfDaysLeft.ts:17]: I think you should have defined a documentation comment at the top of the file for one thing.

**Review by apurvkhare — APPROVED** [review]
LGTM

**Review by vinit717 — DISMISSED** [review]

**Review by sahsisunny — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]

**Review by apurvkhare — APPROVED** [review]

**Pull request merged**